src/main/resources/templates/db_scripts/migration/transform_location_ids_array.sql

```diff
@@ -0,0 +1,8 @@
+-- Transform locationIds arrays to array of objects with locationId field inside them
+UPDATE ${myuniversity}_${mymodule}.fund
+SET jsonb = jsonb - 'locationIds' || jsonb_build_object(
+  'locations',
+  (SELECT jsonb_agg(jsonb_build_object('locationId', value)) from jsonb_array_elements(jsonb->'locationIds'))
+  )
+WHERE jsonb ? 'locationIds'
+AND jsonb->>'locationIds' != '[]';
```

Comment:
I just created new fund via UI with not populating locations and in DB see this structure:
`{
  "id": "9d3dae57-139d-4c55-90ef-66308c16cfd5",
  "code": "TestFund",
  "name": "TestFund",
  "_version": 1,
  "ledgerId": "7cef8378-7cbd-1fae-bcdd-8b9d7c0af9de",
  "metadata": {
    "createdDate": "2024-04-08T10:47:28.909Z",
    "updatedDate": "2024-04-08T10:47:28.909Z",
    "createdByUserId": "4614b2f7-85e2-4243-a487-50d6e5abf14e",
    "updatedByUserId": "4614b2f7-85e2-4243-a487-50d6e5abf14e"
  },
  "acqUnitIds": [],
  "fundStatus": "Active",
  "locationIds": [],
  "allocatedToIds": [],
  "allocatedFromIds": [],
  "externalAccountNo": "12345",
  "restrictByLocations": false,
  "donorOrganizationIds": []
}`
So it has empty array of locations and this script will don nothing with it because of this AND filtering.
I think expected result will be if we have empty array of location objects in this case

Reply:
fixed

Reply:
seems forget to push because could not see this change in schema.json

Reply:
SQL file was updated.